common/test/acceptance/tests/discussion/test_discussion.py

```diff
@@ -425,6 +432,21 @@ def test_discussion_blackout_period(self):
         self.assertFalse(self.thread_page.is_comment_deletable("comment1"))
         self.assertFalse(self.thread_page.is_comment_deletable("comment2"))
 
+    def test_dual_discussion_module(self):
+        self.discussion_page.wait_for_page()
```

Comment:
Our best practice is that Bok Choy test points have a BDD description of what they are testing. For example, https://github.com/open-craft/edx-platform/blob/master/common/test/acceptance/tests/studio/test_studio_split_test.py#L411

Please share this with the whole Escalation team.